termenv_test.go

```diff
@@ -160,6 +162,39 @@ func TestANSIProfile(t *testing.T) {
 	}
 }
 
+func TestWSLDetectioninWindowsTerminal(t *testing.T) {
+	file, err := os.Open("/proc/sys/kernel/osrelease")
+	defer file.Close()
+	if err != nil && !errors.Is(err, os.ErrNotExist) {
+		t.Error("Error reading file proc/sys/kernel/osrelease")
+	}
+
+	tmpFile, err := os.CreateTemp("", "osrelease")
+	defer tmpFile.Close()
+	if err != nil {
+		t.Error("Error making tmp file")
+	}
+
+	io.Copy(tmpFile, file)
+	_, err = io.WriteString(tmpFile, "6.2.6-76060206-microsoft")
+	if err != nil {
+		t.Error("Error writing to tmp file")
+	}
+
+	content, err := os.ReadFile(tmpFile.Name())
```

Comment:
Is writing and reading this temp file really helpful for this test?

Reply:
I don't mean to overwrite `/proc/sys/kernel/osrelease` file.

Reply:
Agreed, that would be silly. As it stands, though, the test doesn't really verify the implementation. Maybe the implementation should be moved to a separate function that expects a variable pointing it to the `osrelease` file?

Reply:
What do you suggest?

Reply:
Move https://github.com/muesli/termenv/pull/141/files#diff-1669ddbefceec37a147d59d67808df4e7c04946f7fee1b79ef838859340fdc14R63-R78 into a separate function that expects a filename instead of operating on a fixed path. This code can then be called and verified from both the actual implementation and a test.

Reply:
Done